src/core/Akka.Remote.Tests.MultiNode/TestConductor/TestConductorSpec.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Actor.Dsl;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using Akka.Remote.Transport;
+using Akka.TestKit;
+
+namespace Akka.Remote.Tests.MultiNode.TestConductor
```

Comment:
Tests the internal behavior of the `TestConductor` to ensure that `Throttling` works - **but** it also has the important implicit trait of checking to see if the backoff behavior within the `EndpointWriter` is correct.